src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -522,6 +522,21 @@
     <value>Exception occurred with following context:
 {0}</value>
   </data>
+  <data name="AnonymousTypeMemberAndNamesCountMismatch2" xml:space="preserve">
```

Comment:
There was pushback from the compiler team on that naming convention in a previous PR.